serverless/components
# Adds Amazon VPC Component

## What has been implemented?

Closes #266 

## Steps to verify

Setup without any inputs should create a VPC with CIDR '10.0.0.0/16', amazonProvidedIpv6CidrBlock 'false' and instanceTenancy default.

```yaml
type: vpc-service

components:
  myAwsVpc:
    type: aws-vpc
```

Following setup should create a VPC with CIDR '10.0.0.0/20', amazonProvidedIpv6CidrBlock 'true' and instanceTenancy "dedicated".

```yaml
type: vpc-service

components:
  myAwsVpc:
    type: aws-vpc
    inputs:
      cidrBlock: 10.0.0.0/20
      amazonProvidedIpv6CidrBlock: true
      instanceTenancy: dedicated
```

## Todos:
